etc/kayobe/environments/aufn-ceph/tenks.yml

```diff
@@ -34,7 +34,7 @@ node_types:
     volumes:
       # There is a minimum disk space capacity requirement of 4GiB when using Ironic Python Agent:
       # https://github.com/openstack/ironic-python-agent/blob/master/ironic_python_agent/utils.py#L290
-      - capacity: 10GiB
+      - capacity: 20GiB
```

Comment:
Combined storage requirements of all VMs is 220GB. SMS lab flavor says bare metal disks are 200GB, Matt checked one and found 233GB.

I'm not sure if these volumes are sparse, but if not they would consume the entire bare metal disk.